From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jbicha@ubuntu.com>
Date: Thu, 11 Dec 2025 15:15:46 -0500
Subject: Adjust rust dependencies to match what is packaged in Debian

---
 tools/html2rss/Cargo.toml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: paperboy/tools/html2rss/Cargo.toml
===================================================================
--- paperboy.orig/tools/html2rss/Cargo.toml
+++ paperboy/tools/html2rss/Cargo.toml
@@ -8,10 +8,10 @@ description = "Generate RSS feed from ar
 
 [dependencies]
 clap = { version = "4", features = ["derive"] }
-reqwest = { version = "0.11", features = ["blocking", "rustls-tls"] }
-scraper = "0.14"
+reqwest = { version = "0.12", features = ["blocking", "rustls-tls"] }
+scraper = "0.20"
 serde_json = "1.0"
-quick-xml = "0.29"
+quick-xml = "0.36"
 url = "2"
 regex = "1"
 rand = "0.8"
